nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -132,6 +141,9 @@ If this is checked, then the binaries will be built and pushed to chacra even if
       - nfs-ganesha-debian
 
     builders:
+      - inject:
+          properties-content: |
+            CEPH_BUILD={scenario}
```

Comment:
@andrewschoen is this redundant or preferred over setting {scenario} as the default for $CEPH_BUILD?